errors/firewall.md

```diff
@@ -29,6 +29,6 @@ If you use Portainer to run the container and get the error:
 Perhaps iptables or your kernel needs to be upgraded.: exit status 3
 ```
 
-This might be because Portainer does not set the `--cap_add=NET_ADMIN` successfully.
```

Comment:
It is `cap_add`, see https://docs.docker.com/reference/compose-file/services/#cap_add